src/components/ToggleMenu/ToggleItem/ToggleItem.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React from 'react';
+import './ToggleItem.css';
+
+interface ToggleItemProp {
+  title: string;
+  handleClick: (e: React.SyntheticEvent) => void;
+  active: string;
+}
+const ToggleItem = (props: ToggleItemProp) => {
+  const { title, handleClick, active } = props;
+  return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events,
```

Comment:
Let's try avoid lint breakers.
In this case you should add tag `<a href="#" >{title}</a>` and add handlers on it